ingestion/src/metadata/profiler/interface/sqlalchemy/bigquery/profiler_interface.py

```diff
@@ -43,12 +76,18 @@ def _get_struct_columns(self, columns: dict, parent: str):
 
     def _get_sampler(self, **kwargs):
         """get sampler object"""
-        session = kwargs.get("session")
```

Comment:
why did you remove this?

Reply:
actually, just found out a method to do overriding, we won't need to remove this. will add in new commit